packages/react-button/src/components/Button/useButton.ts

```diff
@@ -1,11 +1,9 @@
-import { ButtonProps, ButtonSlots, ButtonSlotProps } from './Button.types';
-import { mergeProps } from '../../utils/mergeProps';
+import { ButtonProps } from './Button.types';
 import { ComposePreparedOptions } from '@fluentui/react-compose';
 
 /**
  * The useButton hook processes the Button component props and returns
  * state, slots, and slotProps for consumption by the component.
  * @param props
  */
-export const useButton = (props: ButtonProps, options: ComposePreparedOptions) =>
-  mergeProps<ButtonProps, ButtonSlots, ButtonSlotProps>(props, options);
+export const useButton = (props: ButtonProps, options: ComposePreparedOptions) => options.resolve(props);
```

Comment:
Nothing buttonish about this hook anymore :D

Reply:
leaving it there for future updates.